app/editor/src/components/form/timeinput/TimeInput.tsx

```diff
@@ -0,0 +1,17 @@
+import { InputHTMLAttributes } from 'react';
+
+import * as styled from './styled';
+
+/** Component that will enforce the HH:MM:SS time format */
+export const TimeInput: React.FC<InputHTMLAttributes<HTMLInputElement>> = (props) => {
+  const formatTime = (value: string) => {
```

Comment:
Probably should be a util.  Don't change though.  Not sure what benefit it would have to do make it a util atm.